### DISCUSSION TITLE
Custom MiniDexed PCB (OSH)

### DISCUSSION THREAD

**Discussion by handeyeco** [probonopd/MiniDexed#699] [Show and tell]
Hello 👋

I made a PCB for MiniDexed, and I put the schematics and build files online for anyone interested: https://github.com/handeyeco/minidexed-pcb

Project goals:

- I didn't want to use a bunch of modules
- I wanted the PCB fab house to do the SMT assembly
- I wanted hardware MIDI (in/thru/out)
- I wanted it to be small (it's about the size of a Korg NTS-1)

I just assembled the first one yesterday, but I'm really happy with how it turned out: partially because I've never had a PCB work the first time and partially because it sounds lovely. I still need to do work to put it in some kind of enclosure (probably top/bottom acrylic for now).

Anyway, thought I'd share in case it helps anyone. Thanks MiniDexed team! I'm very excited to jam with it.

**Comment by probonopd** [comment]
Very cool project. Have fun!

**Comment by diyelectromusic** [comment]
That looks great! :)
